methods.tex

```diff
@@ -1,14 +1,10 @@
 \section{Methods}
 
-Liquid-fueled system modeling with contemporary reactor physics codes is 
-difficult because most of these codes were developed for analyzing solid-fueled 
-reactors. In fact, world-wide commercial reactors fleet today is represented by 
-systems with solid fuel. Liquid-fueled systems ability to continuously remove 
-fission products and add fissile and/or fertile elements is sthe main challenge 
-for depletion simulations. SaltProc takes into account online separations and 
-feeds using the SERPENT 2 continuous-energy Monte Carlo neutron transport and 
-depletion code.  All calculations presented in this study were performed using 
-SERPENT 2 version 2.1.30 on Blue Waters’ XE6 nodes. 
+Liquid-fueled systems ability to continuously remove fission products and add 
+fissile and/or fertile elements is the main challenge for depletion simulations. 
+Introduced in this work Python-package, SaltProc, takes into account online 
```

Comment:
"Introduced in this work Python-package, SaltProc," should be "The python package introduced in this work, Salt Proc"